MusicTaste/MusicTaste/MatchingAlgorithm.h

```diff
@@ -8,7 +8,10 @@ NS_ASSUME_NONNULL_BEGIN
 @property (assign, nonatomic) double spotifyMatch;
 + (instancetype)shared;
 - (void) lookForMatches;
+<<<<<<< Updated upstream
 - (void) compareSpotifyData:(PFObject *)potentialMatch withData:(NSManagedObject *)userSpotify;
+=======
```

Comment:
I don't know why this happened but when committing my changes it manually putt these lines in my code. Do you have any idea what these are?

Reply:
Yes, these are git conflict markers https://stackoverflow.com/questions/16389272/git-how-to-remove-conflict-markers-from-files

When you unstash changes that were previously stashed, something might have changed from the time that you stashed that makes the stashed changes incompatible with the new state of the code, or at least git is unsure how to handle the situation. (Maybe you changed the line right above or below; git wants to make sure things are still ok.) So these markers flag those situations for you so you can resolve the conflict. You can either manually remove these markers, or you can use tools built into GitKraken or similar to compare changes and resolve that way.

It's basically saying 

```
<<<<<<<
code from version A
=======
code from version B
>>>>>>>
```

Since there is no code in version B, you can probably just remove the markers and you'll be done, but you'll have to check manually when this happens.